src/docker/DockerApi.ts

```diff
@@ -243,7 +243,8 @@ class DockerApi {
         tarballFilePath: string,
         buildLogs: BuildLog,
         envVars: IAppEnvVar[],
-        registryConfig: DockerRegistryConfig
+        registryConfig: DockerRegistryConfig,
+        overrideOptions: string[] | undefined
```

Comment:
Must be renamed to `dockerBuildOverrideOptions`

Reply:
Thanks, will do.